test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/CodeGenerationTests.cs

```diff
@@ -113,9 +119,37 @@ public void RazorEngine_ModelExpressionTagHelper_DesignTime()
         {
             RunDesignTimeTest("ModelExpressionTagHelper");
         }
+
+        [Fact]
+        public void RazorEngine_RazorPages_DesignTime()
+        {
+            RunDesignTimeTest("RazorPages", BuildDescriptors());
+        }
         #endregion
 
-        private static void RunRuntimeTest(string testName)
+        private IEnumerable<TagHelperDescriptor> BuildDescriptors()
```

Comment:
Both these guys could be static. Can also acquire pranav points on this guy if you want 😄